docs/docs/feature/underglow.md

```diff
@@ -87,7 +87,7 @@ Here's an example of an nRF52 SPI definition:
 :::info
 
 If you are configuring SPI for an nRF52840 (or other nRF52) based board, double check that you are using pins that aren't restricted to low frequency I/O.
-Ignoring these restrictions may result in poor wireless performance. You can find the list of low frequency I/O pins [here](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf52840%2Fpin.html&cp=4_0_0_6_0).
+Ignoring these restrictions may result in poor wireless performance. You can find the list of low frequency I/O pins [here](https://infocenter.nordicsemi.com/index.jsp?topic=%2Fps_nrf52840%2Fpin.html&kp=4_0_0_6_0).
```

Comment:
I'm thinking you haven't.

Reply:
That said, the URL still works 🥇

Reply:
Reverted. Over ambitious VS Code Search and Replace. Good catch.